code/modules/hydroponics/grown/garlic.dm

```diff
@@ -20,3 +21,47 @@
 	bitesize_mod = 2
 	tastes = list("garlic" = 1)
 	wine_power = 10
+
+/obj/item/seeds/garlic/hairy
+	name = "pack of hair garlic"
+	desc = "A packet of extremely hairy seeds."
+	icon_state = "seed-garlic"
+	species = "garlic"
+	plantname = "Garlic Sprouts"
+	product = /obj/item/reagent_containers/food/snacks/grown/garlic/hairy
+	yield = 6
+	potency = 25
+	growthstages = 3
+	growing_icon = 'icons/obj/hydroponics/growing_vegetables.dmi'
+	mutatelist = list(/obj/item/seeds/garlic/hairy)
+	reagents_add = list(/datum/reagent/consumable/garlic = 0.15, /datum/reagent/consumable/nutriment = 0.1)
+
+/obj/item/reagent_containers/food/snacks/grown/garlic/hairy
+	seed = /obj/item/seeds/garlic/hairy
+	name = "hairy garlic"
+	desc = "Delicious, but with a potentially overwhelming odor AND disgusting hair on it like you dropped it on the floor and didn't meet the 5 second rule in time."
+	icon_state = "hairygarlic"
+	tastes = list("garlic" = 1, "hair" = 3)
+	wine_power = 5
+
+/obj/item/reagent_containers/food/snacks/grown/garlic/hairy/Initialize(mapload, obj/item/seeds/new_seed)
+	. = ..()
+	bitesize = reagents.total_volume //one bite!
```

Comment:
make sure that the order of operations is correct with this and the densified chems trait

Reply:
made a workaround